build.sbt

```diff
@@ -207,6 +207,14 @@ val rss = application("rss")
   .aggregate(common)
 
 val main = root()
+// This evicts the version of
+// "com.fasterxml.jackson.core:jackson-databind"
+// used by "com.typesafe.play:sbt-plugin"
+  .settings(
```

Comment:
I'm not too familiar with how Frontend dependencies are set up, but is this the right place to add this? Is there a single place we can track all the dependencies we've "evicted"?

Reply:
hmm, that's a good question. I'm not too sure about previous evictions.

Given each "project" has different dependencies it needs to evict, I think adding them on a "per project" level makes sense. In addition to a comment, it might be useful to refactor the evicted dependencies to a named variable - what do you think?